Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -500,7 +500,56 @@ namespace AzToolsFramework
                 AZStd::string actionIdentifier = "o3de.action.prefabs.detach";
                 AzToolsFramework::ActionProperties actionProperties;
                 actionProperties.m_name = "Detach Prefab";
-                actionProperties.m_description = "Creates a prefab out of the currently selected entities.";
+                actionProperties.m_description = "Selected prefab is detached from its prefab file and becomes normal entities instead.";
```

Comment:
Would prefer if this was a single menu item, and the behavior changed based on a setting in the Editor Preferences.

Reply:
I'll go look at how editor preferences work *how hard could it be*

Reply:
I mean, it's not hard, it's just all hardcoded...
It's in my very long list of things I'd like to rework in the future lol.

Reply:
I'm going to break the existing pattern a bit and put the new setting in the settings registry instead.  So that it doesn't make things even worse by storing even more in the old gSettings / windows registry.